src/components/Editor/Runners/PythonRunner/OutputViewToggle.scss

```diff
@@ -3,8 +3,9 @@
 
 .output-view-toggle {
   display: inline-flex;
-  margin: auto $space-0-5 auto auto;
+  margin-right: $space-0-5;
```

Comment:
I would use `margin-inline-end` as a logical property to make it rtl proof out of box, it will nicely flip sides when language is changed, [Margin examples](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values/Margins_borders_padding#margin_examples)